### PR TITLE
[#186] multi output regression

### PR DESCRIPTION
**Pull request by Samuel-Campbell**
1) Added multi output regressor wrapper
2) will be easier to make many predictions at once
3) NEED TO UPLOAD NEW MODELS ONCE MERGED ** NOT TO SELF

Every column  has a label. From this label we know if we expect an int or boolean value. Iterate each column and if the classifier determined the outcome was present then calculate its integer value.

### CODE REVIEW

**Review by arekmano — COMMENTED** [review]
This change will conflict with #327

File: src/ml_service/model_training/regression/multi_output_regression.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        2) If 1) is true, then replace the value in the vector by it's true integer
+           value.
+
+        Additional notes: step 2) only applies when the boolean value of the classifier
+                          returns True. The reason is that the regressor is trained on biased
+                          inputs. In a sense, the classifier serves a filtering layer to know
+                          which fields are needed by the regressor.
+
+        :param data: np.array([1, 0, 1, 1, ...])
```
Comment (TaimoorRana) [inline src/ml_service/model_training/regression/multi_output_regression.py]: need to update

File: src/ml_service/model_training/regression/multi_output_regression.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        :return: np.array([1, 0, 22, 2, ...])
+        """
+        for i in range(len(outcomes)):
+            if outcomes[i] == 1:
+                column_name = MultiOutputRegression.classifier_labels[i][0]
+
+                if column_name == 'additional_indemnity_date':
+                    pass
```
Comment (Taimoorrana1) [inline src/ml_service/model_training/regression/multi_output_regression.py]: why is this needed?

**Review by arekmano — APPROVED** [review]
Looks good. I don't think we have a case there the data is not biased on win/loss so this approach would definitely work for all the project predictions

**Review by dodoels — APPROVED** [review]
